libraries/lib-music-information-retrieval/MirTypes.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <optional>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace MIR
+{
+enum class FalsePositiveTolerance
+{
+   Strict,
+   Lenient,
+};
+
+enum class TimeSignature
+{
+   TwoTwo,
+   FourFour,
+   ThreeFour,
+   SixEight,
+   _count
+};
+
+struct MusicalMeter
+{
+   const double bpm;
+   const std::optional<TimeSignature> timeSignature;
+};
+
+// Maps a combination of time-signature and number of bars (encoded as
+// string) to a score.
+struct OnsetQuantization
```

Comment:
Not clear in context here, what string you mean.

Use `//!` instead of `//` and it becomes a Doxygen brief comment.

Reply:
That's an outdated comment - thanks.